Mimir/src/models/PlayerStat.php

```diff
@@ -254,12 +301,19 @@ protected function _getOutcomeSummary(int $playerId, array $rounds)
             }
             return $acc;
         }, [
-            'ron'       => 0,
-            'tsumo'     => 0,
-            'chombo'    => 0,
-            'feed'      => 0,
-            'tsumofeed' => 0,
-            'openhand'  => 0
+            'ron'                     => 0,
+            'tsumo'                   => 0,
+            'chombo'                  => 0,
+            'feed'                    => 0,
+            'tsumofeed'               => 0,
+            'openhand'                => 0, // maybe rename it to wins_with_open?
+            'wins_with_riichi'        => 0, // must be equal to riichi_won below
```

Comment:
Why do we need any additional field if there is already one?

Reply:
Looks like they are not always equal, I'll edit the comment.

riichi_won / riichi_lost below are for won / lost sticks that player bets.

In case of double ron and specific rules the stick can be lost and it is counted as riichi_lost, even if the player won in that round.

But I think such wins should be included into wins_with_riichi.